tcf_website/urls.py

```diff
@@ -11,7 +11,10 @@
     path('terms', views.terms, name='terms'),
     path('browse', views.browse, name='browse'),
     path('department/<int:dept_id>', views.department, name='department'),
-    path('course/<int:course_id>', views.course_view_legacy, name='course_legacy'),
```

Comment:
Same comment as before with the changing of the format

Reply:
ahh yeah you're right this happened last time too it's atom autoformatting I'll revert these changes

Reply:
@khushic I've done it for you this time, but in the future when you address a reviewer's comments don't forget to resolve the conversation!